bciers/apps/registration/.eslintrc.json

```diff
@@ -8,10 +8,13 @@
   "globals": { "vi": true },
   "ignorePatterns": ["!**/*", ".next/**/*"],
   "rules": {
+    "react/react-in-jsx-scope": "off",
+    "react/jsx-uses-react": "off",
```

Comment:
This was added to remove a warning from the nx merge airbnb rules. Basically it gives an error if you use jsx in a file without importing React at the top of each file 🤮

Reply:
Okay I actually removed this since I rebased on Josh's PR which fixed these eslint rules!